### PR TITLE
CRM-21599: Added missing structure in templates

### PR DESCRIPTION
**Pull request by mukeshcompucorp**
Overview
----------------------------------------
This PR
- Adds missing HTML structure at `templates/CRM/Admin/Page/LocationType.tpl`
- Adds missing label `Event Name` of event at `templates/CRM/Event/Form/SearchEvent.tpl`

Before
----------------------------------------
before

After
----------------------------------------
after

Technical Details
----------------------------------------
Tabing and spacings are improved

---

 * [CRM-21599: Add missing structure in templates](https://issues.civicrm.org/jira/browse/CRM-21599)

### CODE REVIEW

File: templates/CRM/Event/Form/SearchEvent.tpl [hunk 1]
```diff
@@ -27,11 +27,13 @@
  <h3>{ts}Find Events{/ts}</h3>
   <table class="form-layout">
     <tr class="crm-event-searchevent-form-block-title">
-        <td>{$form.title.html|crmAddClass:twenty}
-             <div class="description font-italic">
-                    {ts}Complete OR partial Event name.{/ts}
-             </div>
-             <div style="height: auto; vertical-align: bottom">{$form.eventsByDates.html}</div>
+        <td>
+          <label>{ts}Event Nanme{/ts}</label>
```
Comment (mattwire) [inline templates/CRM/Event/Form/SearchEvent.tpl]: Typo - should be "Event Name".  Can you get this from $form.title.label?

Comment (mukeshcompucorp) [reply templates/CRM/Event/Form/SearchEvent.tpl]: @mattwire Fixed at https://github.com/civicrm/civicrm-core/pull/11457/commits/92164dec82f17a024f9f9698469e3bce55cbc444

File: templates/CRM/Admin/Page/LocationType.tpl [hunk 2]
```diff
@@ -72,4 +73,5 @@
     {crmButton q="action=add&reset=1" id="newLocationType" icon="plus-circle"}{ts}Add Option{/ts}{/crmButton}
     {crmButton p="civicrm/admin" q="reset=1" class="cancel" icon="times"}{ts}Done{/ts}{/crmButton}
   </div>
+</div>
```
Comment (pradpnayak) [inline templates/CRM/Admin/Page/LocationType.tpl]: How is this file related to field name change?

**Comment by pradpnayak** [comment]
I am not sure if its a great idea to have label for the text field since there is enough description about the usage of the field. I will this to core team to decide. 

I have tested this locally and it works as expected. However, I am not confident of having 'templates/CRM/Admin/Page/LocationType.tpl' file included in this PR. Is it some how related to the fix? And also can you squash the commits?

File: templates/CRM/Admin/Page/LocationType.tpl (hunk 2 above)
Comment (mukeshcompucorp) [reply templates/CRM/Admin/Page/LocationType.tpl]: @pradpnayak This is not just for adding label. PR is to make the structure proper and this file adds missing wrapper around the content like other files.

Comment (pradpnayak) [reply templates/CRM/Admin/Page/LocationType.tpl]: :)

Comment (mukeshcompucorp) [reply templates/CRM/Admin/Page/LocationType.tpl]: Should I understand :) as approved then?

**Comment by mukeshcompucorp** [comment]
@pradpnayak 
to your question
> I am not sure if its a great idea to have label for the text field since there is enough description about the usage of the field. I will this to core team to decide.

Description should always depict more about field but, it is the label that ultimately tells people what the field is for.
That's why WCAG standards says to have a label for each input,
so, label -> field name/title
description -> more about what you can do to the field

File: templates/CRM/Admin/Page/LocationType.tpl (hunk 2 above)
Comment (pradpnayak) [reply templates/CRM/Admin/Page/LocationType.tpl]: Yes APPROVED.

Comment (mukeshcompucorp) [reply templates/CRM/Admin/Page/LocationType.tpl]: @pradpnayak can you do that from above Review changes then? for others to know the status

**Comment by monishdeb** [comment]
I am not opposed to these change but like @pradpnayak said we already had an inline description for this field. But still, if we need to show the label then we also need to made such changes to other Search forms too like:
1. Manage Group screen: screen shot 2017-12-27 at 8 24 04 pm
2. Manage Contribution page: screen shot 2017-12-27 at 8 23 22 pm

@mukeshcompucorp would you like to discuss the PR on our dev channel https://chat.civicrm.org/civicrm/channels/dev ?

**Comment by monishdeb** [comment]
Tested the PR and works for me. Doesn't bring any unintended change for the additional fixes made in localtionType.tpl. But next time please bring the indentation fixes or any NFC changes in separate PR.

Manage Event page after applying this patch:
screen shot 2017-12-28 at 3 42 40 pm
